File: test_data/grammar_samples/sample.py
# Python sample
import os
from typing import List

def fibonacci(n: int) -> List[int]:
    """Return first n Fibonacci numbers."""
    result = [0, 1]
    for i in range(2, n):
        result.append(result[-1] + result[-2])
    return result

class Counter:
    def __init__(self, start: int = 0):
        self.value = start

if __name__ == "__main__":
    print(fibonacci(10))

File: test_data/grammar_samples/sample.java
// Java sample
import java.util.List;
import java.util.ArrayList;

public class Sample {
    private final String name;

    public Sample(String name) {
        this.name = name;
    }

    public static void main(String[] args) {
        List<Integer> numbers = new ArrayList<>();
        numbers.add(42);
        System.out.println("Count: " + numbers.size());
    }
}
